### DISCUSSION TITLE
Doc Improvements

### DISCUSSION THREAD

**Discussion by chrisgrieser** [Taitava/obsidian-shellcommands#156] [Ideas]
Since you mentioned that you are not that active on Discord, I wanted to forward you this offer – there is a technical writer offering to help with plugin documentation, and you might wanna take them up on their offer.

https://discord.com/channels/686053708261228577/707816848615407697/942882658252382218

**Comment by Taitava** [comment]
Thank you for letting me know about this! 😎 I contacted them, so let's see what happens. 🙂👍
